bucket/tic-80.json

```
@@ -1,10 +1,10 @@
 {
-    "homepage": "https://tic.computer/",
-    "description": "Fantasy computer for making, playing and sharing tiny games",
-    "version": "0.70.6",
+    "homepage": "https://tic80.com",
+    "description": "TIC-80 is a fantasy computer for making, playing and sharing tiny games.",
```

We don't end descriptions with a period:

```suggestion
    "description": "TIC-80 is a fantasy computer for making, playing and sharing tiny games",
```